app/assets/v2/css/results.css

```diff
@@ -0,0 +1,222 @@
+
+/* Results */
+
+.results h5 {
+  font-weight: 700;
```

Comment:
thoughts on adding class `font-weight-bold` to h5 as opposed to adding it here ?